DDDEastAnglia/Views/Shared/_Sponsors.cshtml

```diff
@@ -1,63 +1,11 @@
-﻿<section id="sponsors">
+﻿@model IEnumerable<DDDEastAnglia.Models.SponsorModel>
+<section id="sponsors">
```

Comment:
There were subtle differences in the markup.. I think with some css foo it should be possible.